lib/options.go

```diff
@@ -388,6 +388,9 @@ type Options struct {
 
 	// Redirect console logging to a file
 	ConsoleOutput null.String `json:"-" envconfig:"K6_CONSOLE_OUTPUT"`
+
+	// Specify client IP ranges and/or CIDR from which VUs will make requests
+	LocalIPs types.NullIPPool `json:"-" envconfig:"K6_LOCAL_IPS"`
```

Comment:
I am not sure why this shouldn't be configured via the JS `options` and from `config.json`. We can validate so we don't allow it in the cloud, but I don't see to disable it in general.

Reply:
Why should it be configurable? I find it very unlikely that will be a needed thing for anyone and setting an environment variable won't be a good enough solution as well. 

I am for leaving this for when someone actually asks for it as we can always add it

Reply:
> Why should it be configurable?

Because there's no good reason not to be? And, whenever possible, most things in k6 are configurable in 4 different ways - CLI flags, env vars, JS options, JSON options. When that's not the case, it's usually because there is a good reason for it. For example, we need to know the option's value _before_ we execute the init context. Or we think the option is so minor that it'll overcrowd the CLI flags. Or there's a bug in `envconfig`, or something like that.

The more inconsistencies we have with the k6 options, the worse the user experience is. It's not a big deal, but constantly breaking expectations in this way is bound to get annoying.